lib/rails_failover/active_record/railtie.rb

```diff
@@ -11,7 +11,7 @@ class Railtie < ::Rails::Railtie
         app.config.active_record_rails_failover = true
         ::ActiveSupport.on_load(:active_record) do
           begin
-            ::ActiveRecord::Base.connection.verify!
+            ::ActiveRecord::Base.connection.tap(&:verify!).active?
```

Comment:
It also seems like just calling `verify!` is not sufficient since `verify!` only connects to the PG server but does not actually make a SQL query.